res/css/views/right_panel/_UserInfo.scss

```diff
@@ -76,33 +76,12 @@ limitations under the License.
         width: 160px;
     }
 
-    .mx_UserInfo_avatar {
-        margin: 24px 32px 0 32px;
-    }
-
-    .mx_UserInfo_avatar > div {
-        max-width: 30vh;
-        margin: 0 auto;
-        transition: 0.5s;
```

Comment:
this transition you got rid of was for the size of the avatar shrinking

Reply:
It also doesn't shrink anymore

Before
![image](https://user-images.githubusercontent.com/2403652/120667444-68a0df80-c485-11eb-8c3b-05f981a21da8.png)


After
![image](https://user-images.githubusercontent.com/2403652/120667427-6474c200-c485-11eb-9d74-d1af35c84ccc.png)